events/views.py

```diff
@@ -86,14 +90,49 @@ def get_queryset(self):
 
 
 class AttendeePaymentView(generic.View):
-    template_name = 'main/payment.html'
 
     def get(self, request, *args, **kwargs):
         if services.EventService().count(kwargs.get('pk')):
             pk = self.kwargs.get('pk')
             event = models.Event.objects.get(pk=pk)
-            payment = event.price
-            return render(request, self.template_name, {'payment': payment})
+            if self.request.user.profile.stripe_access_token is None and self.request.user.profile.stripe_user_id is None:
+                return redirect('authorize')
+            else:
+                if event.has_finished:
+                    attende_list = selectors.UserSelector().event_attendees(pk)
+
+                    for attendee in attende_list:
+
+                        transaction = models.Transaction.objects.filter(created_by=attendee, event=event).first()
+
+                        if transaction is not None:
```

Comment:
Es una tontería pero ya que vas a cambiar cosas puedes también poner `if transaction` (comprueba si la variable está llena o es None, no te hace falta el is not None). Puedes aplicarlo en más lugares. No es necesario pero si quires puedes cambiarlo.